WcaOnRails/spec/support/test_db_manager.rb

```diff
@@ -0,0 +1,16 @@
+class TestDbManager
+  CONSTANT_TABLES = %w(Countries Continents Events Rounds Formats teams).freeze
+
+  def self.fill_tables
+    Dir["#{Rails.root}/db/seeds/*.seeds.rb"].each { |file| load file }
+  end
+end
+
+describe TestDbManager do
+  it "CONSTANT_TABLES includes all tables filled in the files inside /db/seeds/ directory" do
+    expected_files = TestDbManager::CONSTANT_TABLES.map do |table_name|
+      "#{Rails.root}/db/seeds/#{table_name.underscore}.seeds.rb"
```

Comment:
I don't think you need the `#{Rails.root}/` part(s). That should always be the current directory

Otherwise LGTM.